packages/@aws-cdk/aws-appmesh/lib/tls-validation-context.ts

```diff
@@ -0,0 +1,160 @@
+import * as acmpca from '@aws-cdk/aws-acmpca';
+import { CfnVirtualGateway, CfnVirtualNode } from './appmesh.generated';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct } from '@aws-cdk/core';
+
+/**
+ * Represents the properties needed to define TLS validation context
+ */
+export interface TlsValidationContext {
+  /**
+   * Reference to where to retrieve the trust chain.
+   */
+  readonly trust: TlsValidationContextTrust;
+}
```

Comment:
This interface is redundant. I would remove it.

Reply:
Please refer [this response](https://github.com/aws/aws-cdk/pull/14782#discussion_r636273194) and advise if it makes to retain `TlsValidationContext` interface.